src/harness/harness.ts

```diff
@@ -1623,10 +1622,10 @@ namespace Harness {
         }
     }
 
-    // Regex for check if the give filePath is a library file
-    const libRegex = /lib(\.\S+)*\.d\.ts/;
-    export function isLibraryFile(filePath: string): boolean {
-        return !!libRegex.exec(Path.getFileName(filePath));
+    export function isDefaultLibraryFile(filePath: string): boolean {
+        // We need to make sure that the filePath is prefix with "lib." not just containing "lib." and end with ".d.ts"
+        const fileName = Path.getFileName(filePath);
+        return fileName.indexOf("lib.") === 0 && !!fileName.match(/\.d\.ts/);

```

Comment:
+1 why not just add a ^ and $ to the existing regex? Is it broken for other reasons?